languages/en-PH/plugin.info

```diff
@@ -0,0 +1,9 @@
+{
+	"title": "$:/languages/en-PH",
+	"name": "en-PH",
+	"plugin-type": "language",
+	"description": "English (PH)",
+	"author": "JC John Sese Cuneta",
+	"core-version": ">=5.3.5",
+	"dependents": ["$:/languages/en-GB"]
```

Comment:
dependents is not needed for the core language. It will be always there

Reply:
Got it. Thank you!